odoo/http.py

```diff
@@ -788,6 +789,18 @@ def is_method_a_route(cls):
                     _logger.warning("The endpoint %s is not decorated by @route(), decorating it myself.", f'{cls.__module__}.{cls.__name__}.{method_name}')
                     submethod = route()(submethod)
 
+                # TODO move linting/check to a dedicated test.
+                if 'type' in merged_routing:  # Parent was browsed
+                    useless_overrides = {
+                        key: value
+                        for key, value in submethod.original_routing.items()
+                        if key in merged_routing and merged_routing[key] == value
+                        and key not in ('routes', 'type')
+                    }
+                    if useless_overrides and not(len(useless_overrides) == 1 and 'type' in useless_overrides):
+                        _logger.warning("Redefining parent route value for route %s: %s",
+                            ",".join(merged_routing['routes']), pformat(useless_overrides))
+
```

Comment:
Could you keep the same warning style as above? Only for sake of consistency :p

```suggestion
                        _logger.warning("The endpoint %s is duplicating the existing routing configuration : %s",
                            f'{cls.__module__}.{cls.__name__}.{method_name}', pformat(useless_overrides))

```